api/v1alpha1/amaltheasession_types.go

```diff
@@ -312,6 +312,16 @@ type Culling struct {
 	// Golang's time.ParseDuration is used to parse this, so values like 2h5min will work,
 	// valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 	MaxHibernatedDuration metav1.Duration `json:"maxHibernatedDuration,omitempty"`
+	// +optional
+	// +kubebuilder:validation:Format:=date-time
+	// +kubebuilder:validation:Type:=string
+	// A timestamp denoting the time when a user approved to have interacted with the session,
+	// preventing it from culling. This timestamp is used to restart MaxIdleDuration.
```

Comment:
Let's rephrase this:


```suggestion
	// A timestamp denoting the time when a user has proven to have interacted with the session,
	// preventing it from culling.
	// This means that the following invariant is always true: `status.idleSince <= spec.culling.lastInteraction`.
```

Reply:
I updated the comment, hopefully the intention is clear.